packages/components/src/DateTimePickers/Time/Manager/Manager.component.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TimeContext } from '../Context';
+import extractTime from '../time-extraction';
+
+class ContextualManager extends React.Component {
```

Comment:
You can name that `Manager`. Its purpose is to provide the context, no need to add `Contextual` in it.